node/tessel-export.js

```diff
@@ -640,6 +719,29 @@ Tessel.Pin.prototype.analogWrite = function(val) {
   return this;
 };
 
+// Duty cycle should be a value between 0 and 1
+Tessel.Pin.prototype.pwmDutyCycle = function(dutyCycle, cb) {
```

Comment:
This function is a potential hot path, so we need to be prepared to optimize this aggressively.